src/pages/stylePage/ChooseAIStylePage.tsx

```diff
@@ -0,0 +1,205 @@
+import Button from "@components/commons/button/Button";
+import Header from "@components/commons/header/Header";
+import type { ContentsProps, ItemProps } from "@pages/stylePage/types";
+import React, { useEffect, useMemo, useRef, useState } from "react";
+
+import * as S from "./ChooseAIStylePage.styled";
+
+const ChooseAIStylePage: React.FC = () => {
+  // URL 목록은 한 번만 생성
+  const presetUrls = useMemo(
+    () => [
+      {
+        imgaeName: "sample1",
+        url: "https://images.unsplash.com/photo-1508214751196-bcfd4ca60f91?q=80&w=800&auto=format&fit=crop",
+      },
+      {
+        imgaeName: "sample2",
+        url: "https://images.unsplash.com/photo-1512496015851-a90fb38ba796?q=80&w=800&auto=format&fit=crop",
+      },
+      {
+        imgaeName: "sample3",
+        url: "https://images.unsplash.com/photo-1540206276207-3af25c08abc4?q=80&w=800&auto=format&fit=crop",
+      },
+    ],
+    []
+  );
+
+  const defaultItemInfo: ItemProps = useMemo(
+    () => ({ name: "", content: "", category: "" }),
+    []
+  );
```

Comment:
P2) 이 부분도! 객체 재생성 방지 측면에서 `useMemo`를 사용한 것 같은데 그냥 컴포넌트 밖으로 빼서
```suggestion
  const DEFAULT_ITEM_INFO: ItemProps = { name: "", content: "", category: "" };
```
으로 사용하면 될 것 같아요! 대신 이러면 아래 ContentsProps도 변경해 줘야 하는데